### DISCUSSION TITLE
Showing both components the beacon and tooltip at the same time

### DISCUSSION THREAD

**Discussion by DaliaW** [gilbarbara/react-joyride#734] [Q&A] [answered]
Can I have both components displayed at the same time, (the beacon component and the tooltip) instead of showing first the beacon and then showing the tooltip after the user clicks on the beacon?

**Comment by gilbarbara** [comment, marked as answer]
Hey @DaliaW 

That doesn't make much sense since the beacon's function is to open the tooltip.

Maybe do you want to skip the beacon and display the tooltip directly?
Are you setting the [continuous prop](https://docs.react-joyride.com/props) to `true`?
You can disable the first beacon by setting the [disableBeacon](https://docs.react-joyride.com/step) property in the first step.

**Comment by DaliaW** [comment]
Thank you so much for your answer 🙌
I've another question, can I show a step to be displayed based on a condition? For example, if the user clicked on the skip button can I have a step get shown that the user can access the tour again by clicking on a certain button?

**Reply by gilbarbara** [reply]
Nope, the skip terminates the tour.
However, you can listen to the skip action in the callback and display a standalone tooltip with your message. This library uses [react-floater](https://github.com/gilbarbara/react-floater) for its tooltips, so just install it in your dependencies and configure the extra tooltip.
